docs/admin/overview.mdx

```diff
@@ -242,3 +243,18 @@ The Payload Admin Panel is translated in over [30 languages and counting](https:
 ## Light and Dark Modes
 
 Users in the Admin Panel have the ability to choose between light mode and dark mode for their editing experience. Users can select their preferred theme from their account page. Once selected, it is saved to their user's preferences and persisted across sessions and devices. If no theme was selected, the Admin Panel will automatically detect the operation system's theme and use that as the default.
+
+## Timezones
+
+The `admin.timezone` configuration allows you to configure timezone settings for the Admin Panel. You can customise the available list of timezones and in the future configure the default timezone for the Admin Panel and for all users.
+
+The following options are available:
+
+| Option            | Description                                     |
+| ----------------- | ----------------------------------------------- |
+| `options`         | An array of label/value options for selectable timezones where the value is the IANA name eg. `America/Detroit`                        |
+
+<Banner type="info">
+  **Good to know**
+  You can enable timezones on each individual date field. See [Date Fields](../fields/overview#date) for more information.
```

Comment:
Should we do this? This would force us to pass the options through as props to the date field. Not the end of the world, but the date field could otherwise retrieve these for itself from the config.